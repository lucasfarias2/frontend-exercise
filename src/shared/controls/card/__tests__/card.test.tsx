import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Card from '../card';

afterEach(cleanup);

describe('<Card />', () => {
  it('renders correctly', () => {
    const { container } = render(<Card className="test" />);
    expect(container).toMatchSnapshot();
  });
});
