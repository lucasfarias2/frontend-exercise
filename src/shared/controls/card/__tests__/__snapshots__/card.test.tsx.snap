// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders correctly 1`] = `
<div>
  <div
    class="ui__card test"
  />
</div>
`;
